recipeezy/static/recipes/Chicken_Provencal

```diff
@@ -0,0 +1,33 @@
+NAME:
+Classic Chicken Provencal
+CUISINE:
+French
+DESCRIPTION:
+Chicken Provencal is a regional French recipe that can take on many forms. There is no one absolute recipe for Chicken Provencal, but there are some pieces that are integral to the success of the dish. The cut of chicken you use is important, as it is with any braised dish, but also the dish should include some lemons, briny ingredients like olives and/or capers, ripe tomatoes, and lots of herbs. Both dried and fresh herbs can work.
+
+The skillet will have a wonderful sauce that created from the wine, chicken fat, and vegetables, so serving the dish with a crusty baguette or over rice is the only way to go.
+TAGS:
+MEAT, COMFORT FOOD, CHEAP DINNER, FRENCH
+INGREDIENTS:
+7-8 - bone-in, skin-on chicken thighs
+1 tsp kosher salt
+1/2 tsp black pepper
+2 tbs olive oil
+1 tbs Herbes de Provence
+4 large shallots, quartered
+6 cloves garlic, peeled
+1 pint cherry tomatoes
+1/2 cup green olives
+1 - lemon, cut into eighths
+1 tablespoon capers
+4 sprigs thyme
+1 cup white wine
+- - Rice or baguette
+RECIPE:
+1. Preheat Oven to 400°F
+2. Season the chicken thighs well on all side with salt and pepper.
+
```

Comment:
Consider changing the plaintext recipes to JSON objects

Reply:
ya, JSON will be in the next commit